# ¿Cuál es el contenido al que más se hacen redirecciones?

En `redirect_list.csv` tengo la columna `rd_title` que contiene el titulo del contenido al que redige cada fila.

Hallo el titulo que mas se repite en esta columna ya que representa el titulo del contenido al que mas se hacen redirecciones.

In [1]:
# importacion general de librerias y de visualizacion (matplotlib y seaborn)
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

plt.style.use('default') # haciendo los graficos un poco mas bonitos en matplotlib
#plt.rcParams['figure.figsize'] = (20, 10)

sns.set(style="whitegrid") # seteando tipo de grid en seaborn

pd.options.display.float_format = '{:20,.2f}'.format # suprimimos la notacion cientifica en los outputs

import warnings
warnings.filterwarnings('ignore')

from datetime import datetime

In [2]:
# Cargo los datasets
wiki_contents=pd.read_csv('./Wikipedia_dataset_2c2021/redirect_list.csv')

## Evaluo si es necesario hacer una limpieza de datos. 

Si el valor que mas se repite en `rd_title` es `Nan` entonces lo descarto. De lo contrario me quedo con el primero 

In [3]:
wiki_contents['rd_title'].value_counts()

Artículo_futuro                                  1161
IV_milenio                                       1001
V_milenio                                         998
Estaciones_de_Metrobús_de_la_Ciudad_de_México     396
163.117.0.0                                       310
                                                 ... 
Gabriela_(telenovela_brasileña)                     1
Gabriela_(telenovela)                               1
Margarita_Eriksdotter,_reina_de_Noruega             1
Bose_Corporation                                    1
Bercero_(desambiguación)                            1
Name: rd_title, Length: 726321, dtype: int64

In [4]:
wiki_contents['rd_title'].value_counts().nlargest(1)

Artículo_futuro    1161
Name: rd_title, dtype: int64

## Respuesta

El contenido al que mas le hacen redirecciones es `Artículo_futuro`, con 1161 redirecciones.